lib/token.js

```diff
@@ -543,9 +561,63 @@ function getWithoutPrompt(sdk, oauthOptions, options) {
   return getIdToken(sdk, oauthParams, options);
 }
 
+function getWithPopup(sdk, oauthOptions, options) {
+  var oauthParams = util.clone(oauthOptions) || {};
+  util.extend(oauthParams, {
+    display: 'popup'
+  });
+  return getIdToken(sdk, oauthParams, options);
```

Comment:
Should we rename this function to "getToken" given that we can ask for an accessToken?